cac-prod.yml

```diff
@@ -169,6 +169,12 @@ jenkins:
         name: RD
         recurse: true
         title: Ref Data Dashboard
+    - buildMonitor:
+        includeRegex: >-
+          HMCTS_.*IDAM.*\/(idam-api|idam-web-public|idam-web-admin)\/PR.*
```

Comment:
perhaps `idam-*` is enough

Reply:
These three builds currently cannot run in parallel. Whilst that is being addressed, this visibility helps cause less concurrency issues. `idam-*` matches too many projects.

Reply:
@doncem please resolve or continue this discussion.

Reply:
I'm not the owner of repo nor IDAM project - not aware what's needed. I merely reviewed as outsider here giving extra thought 😉